### PR TITLE
Add stop headsigns for flex

### PR DESCRIPTION
**Pull request by philip-cline**
### Checklist

- [x] Appropriate branch selected _(all PRs must first be merged to `dev` before they can be merged to `master`)_
- [x] Any modified or new methods or classes have helpful JSDoc and code is thoroughly commented
- [x] The description lists all applicable issues this PR seeks to resolve
- [x] The description lists any configuration setting(s) that differ from the default settings
- [x] All tests and CI builds passing

### Description

Add stop headsigns for flex now too. GTFS-lib PR for testing: https://github.com/conveyal/gtfs-lib/pull/373

### CODE REVIEW

File: lib/editor/components/pattern/PatternStopCard.js [hunk 1]
```diff
@@ -420,7 +420,7 @@ class PatternStopContents extends Component<Props, State> {
 
   _onHeadsignChange = (e: SyntheticInputEvent<HTMLInputElement>) => {
     const {activePattern, index, updatePatternStops} = this.props
-    const patternStops = [...activePattern.patternStops]
+    const patternStops = this.getPatternHaltsFromActivePattern()
```
Comment (miles-grant-ibigroup) [inline lib/editor/components/pattern/PatternStopCard.js:423]: Does this method exist?

**Review by miles-grant-ibigroup — CHANGES_REQUESTED** [review]
Makes sense in theory but I have a quick question about the implementation

File: lib/editor/components/pattern/PatternStopCard.js (hunk 1 above)
Comment (philip-cline) [reply lib/editor/components/pattern/PatternStopCard.js:423]: Resolved

**Review by miles-grant-ibigroup — APPROVED** [review]
Works well for me thanks for the changes

**Review by binh-dam-ibigroup — APPROVED** [review]
LGTM

**Comment by philip-cline** [comment]
Awaiting backend PR merge

**Pull request merged**